tests/test_utils.py

```diff
@@ -345,3 +357,76 @@ def test_msdf_from_mappings(self):
         self.assertEqual(1, len(new_msdf.df.index))
         self.assertEqual(rows[0], tuple(msdf.df.iloc[0]))
         self.assertEqual(new_msdf.metadata, msdf.metadata)
+
+    def test_get_dict_from_mapping(self):
+        """Test getting dict from a SSSOM mapping object or a dictionary."""
+        mapping_obj = SSSOM_Mapping(
+            subject_id="DOID:0050601",
+            predicate_id="skos:exactMatch",
+            object_id="UMLS:C1863204",
+            mapping_justification=SEMAPV.ManualMappingCuration.value,
+            author_id=["orcid:xxxxxxxx", "orcid:yyyyyyyy"],
```

Comment:
let's use real ORCIDs

Reply:
Changed. It's a test, doesn't really matter.